pkg/postage/postagecontract/contract.go

```diff
@@ -49,6 +49,7 @@ type Interface interface {
 	CreateBatch(ctx context.Context, initialBalance *big.Int, depth uint8, immutable bool, label string) ([]byte, error)
 	TopUpBatch(ctx context.Context, batchID []byte, topupBalance *big.Int) error
 	DiluteBatch(ctx context.Context, batchID []byte, newDepth uint8) error
+	ExpireBatches(ctx context.Context) error
```

Comment:
If we only have to export this single functionality, we should ideally create a new interface
```
type PostageBatchExpirer interface {
        ExpireBatches(context.Context) error
}
```

The storage incentives agent only needs to know about this single interface and not the whole contract.